flake.nix

```
@@ -9,7 +9,7 @@
 
     # for the purescript project
     ogmios.url = "github:mlabs-haskell/ogmios";
-    ogmios-datum-cache.url = "github:mlabs-haskell/ogmios-datum-cache/1be5e903e14f5a1785983bdb38203195b401bd15";
+    ogmios-datum-cache.url = "github:mlabs-haskell/ogmios-datum-cache/85e030ae914abcffba4a7c261d9cd81f98b2168c";
```

```suggestion
    ogmios-datum-cache.url = "github:mlabs-haskell/ogmios-datum-cache/def286f8e0e3c54707ef69578ac71a5e81c708a9";
```
This has control of the `logging` level.